### ISSUE TITLE
Feature request: Ignore generated files by Rails

### ISSUE THREAD

**Issue by frodsan** [testdouble/standard#27]
What do you think about checking if Rails is being used and ignore these files by default:

```
ignore:
  - 'bin/**/*'
  - 'db/schema.rb'
  - 'tmp/**/*'
```

Less config, more drop-in. I can try to make a PR if you think this is useful :)

**Comment by searls** [comment]
This seems entirely reasonable. We could enable/disable the Rails cops based on the same detection.

**Comment by danielmorrison** [comment]
Heroku's [ruby buildpack](https://github.com/heroku/heroku-buildpack-ruby) checks the `Gemfile` for `railties`. They do a lot more with the `Gemfile`, but we could do similar: 

The code could be this, plus some error checking:

```ruby
require "bundler"
gemfile_contents = File.read("./Gemfile.lock")
parser = Bundler::LockfileParser.new(gemfile_contents)
parser.specs.any? {|s| s.name == "railties"}
```

**Comment by searls** [comment]
I plan on implementing this and #42 in tandem with one another

**Comment by searls** [comment]
I just revisited this after implementing #52 to close #42 and found the only proposed ignore rule that isn't enabled by default is actually `db/schema.rb`.

It seems to me that rather than write a potentially complex/slow Rails detection scheme, it makes more sense to just ignore `<config_root>/db/schema.rb`. What are the odds someone will have a real ruby file at exactly that location in a non-Rails app and also really want that file linted? Seems really low.

I propose we just hold our noses and add `'db/schema.rb'` to the default ignores that standard adds.

Are there any other ignore rules we really ought to add? Here's the current list FWIW https://github.com/testdouble/standard/blob/master/lib/standard/creates_config_store/configures_ignored_paths.rb#L3-L13

**Comment by searls** [comment]
Landed in 0.0.19
